### ISSUE TITLE
Defector flickering/drifting/lagging image on Odyssey+

### ISSUE THREAD

**Issue by Tiergott** [LibreVR/Revive#1548]
I'm getting good framerate, but the picture drags behind during head turning and like flickering somehow. I'm using a Samsung Odyssey+. This issue is reproduced only in Defector for now

1. Start Defector
2. Start game and try to turn your head in any direction

Environment:
 - App: Defector
 - OS: Win10
 - Headset: Samsung Odyssey+

Versions:
 - Revive: 2.0.0.13
 - SteamVR: beta 1.12.3

I tried revive 1.8.0.x and this issue was reproduced as well

[SteamVR-2020-05-15-PM_03_56_02.txt](https://github.com/LibreVR/Revive/files/4634481/SteamVR-2020-05-15-PM_03_56_02.txt)

**Comment by hail2theking28** [comment]
Yes,defector for me has really bad stutter,its unplayable right now, I wonder if there will be a fix soon.

**Comment by Megaace23** [comment]
Yes, I have the same problem... :(

**Comment by Tiergott** [comment]
Is there any progress on fixing this?... Still could not play Defector(

**Comment by CrossVR** [comment]
Due to the way this game works it is not possible to fix this problem for WMR headsets using SteamVR. It should work for the Valve Index and HTC Vive though.

ReviveXR has the potential to fix this using OpenXR, but it is not stable enough yet so I doubt it will be able to run Defector.

**Comment by Tiergott** [comment]
Understood, thanks for reply)
